src/components/ToolsUsed.astro

```diff
@@ -0,0 +1,126 @@
+<section class="container max-w-screen-2xl mx-auto my-28">
+  <div class="px-4">
+    <h2 class="text-2xl md:text-4xl font-semibold tracking-wider mb-4 text-center lg:pt-3">
+      Industries we work in
+    </h2>
+    <p class="pb-3 md:pb-6 mb-0 sm:mb-2 lg:mb-4 text-center text-body">
+      Most of our projcts come from the industries below
+    </p>
+    <div class="flex flex-wrap lg:items-center">
+      <!-- ACCORDIAN -->
+      <div class="w-full md:w-1/2 pb-2 lg:pb-0 mb-4 md:mb-0 px-4">
+        <div class="max-w-lg" id="industries">
+          <div class="-mb-3 lg:-mb-2 xl:mb-1">
+            <h3 class="accordion-header text-lg mb-0 font-semibold md:text-2xl active">
+              <button
+                class="relative flex items-center w-full text-left border-none cursor-pointer py-4"
+                type="button"
+              >
+                Startups and technology
```

Comment:
I would suggest adding more space between the titles so it doesn't look as crowded! 😄